distribution/src/main/release/samples/pom.xml

```diff
@@ -99,6 +99,7 @@
         <module>wsdl_first_rpclit</module>
         <module>jms_pubsub</module>
         <module>jax_rs/spring_security</module>
+        <module>jax_rs/spring_boot</module>
```

Comment:
As for me, makes sense to update other samples, @sberyozkin what do you think? Thanks, @johnament !